source/BulkCrapUninstaller/Properties/Localisable.it.resx

```diff
@@ -478,7 +503,8 @@ Vuoi utilizzare i disinstallatori guidati per questi elementi, o preferisci sian
     <value>Il file non esiste. </value>
   </data>
   <data name="PropertiesWindow_Table_ErrorMsi" xml:space="preserve">
-    <value>Niente da mostrare. Questa appicazione si disinstalla con Windows Installer (MsiExec).</value>
+    <value>Nulla da visualizzare. 
+Questa appicazione si disinstalla con Windows Installer (MsiExec).</value>

```

Comment:
Adding new lines like this can cause issues in the interface, in this case the second line might be cut off. Did you try building and testing BCU with the new translations?